modules/amazon-base.nix

```diff
@@ -1,13 +1,12 @@
-{ name, config, resources, ... }:
-
 with (import ./../lib.nix);
 
+{ name, config, resources, ... }:
+
 {
  deployment.targetEnv = "ec2";
  deployment.ec2.instanceType = mkDefault "t2.large";
- deployment.ec2.region = mkDefault region;
- deployment.ec2.keyPair = mkDefault (resources.ec2KeyPairs.${keypairFor config.deployment.ec2.accessKeyId region});
- deployment.ec2.securityGroups = mkDefault [resources.ec2SecurityGroups."allow-all-${config.deployment.ec2.region}" ];
```

Comment:
Removing this breaks Hydra and infrastructure in general, I think.

Reply:
Thanks, I'll look into it!

Reply:
Fixed.